### DISCUSSION TITLE
How to disable acceleration control

### DISCUSSION THREAD

**Discussion by Pippisbruffi** [OrcaSlicer/OrcaSlicer#4945] [General]
I've set my gcode flavour to Klipper and, as expected, enabling or disabling "emit_machine_limits_to_gcode" does nothing. Either way, when I slice an object, I see that some commands like "SET_VELOCITY_LIMIT" are written in the gcode. I don't want that, I want Klipper to manage acceleration by itself. What can I do?

**Comment by vgdh** [comment]
set default acceleration to zero in speed section

**Reply by oligazar** [reply]
There's no default acceleration setting in speed section

**Reply by vgdh** [reply]
You can find it in speed section, scroll down until you see acceleration settings
now days it named "normal printing"

**Reply by afiskon** [reply]
It doesn't seem to do what expected. I tried to follow this article https://www.klipper3d.org/Pressure_Advance.html when I set acceleration control for "normal printing" to zero and my printing speed is 100 mm/s the walls of the model are still printed at ~67 mm/s. I checked other settings including volumetric speed for the nozzle etc. No matter what I do I don't see a way to print the walls at 100 mm/s which I believe is what the test wants. So some smart acceleration control seems to be left.

(I know that Orca has a similar test build-in but I don't like it because one-layer mouse ears the tower has are difficult to remove from my PEI build plate - they are left when I lift the model and I have to scratch the build plate for a long time. No such problem with the model on the Klipper web site.)

**Reply by afiskon** [reply]
<img width="751" alt="Screenshot 2024-12-09 at 23 53 27" src="https://github.com/user-attachments/assets/acfc8f3f-f0db-4e8e-bb81-96874f791276">

**Reply by afiskon** [reply]
Orca version is 2.2.0

**Reply by afiskon** [reply]
Never mind I've found the setting that slowed everything down!
<img width="748" alt="Screenshot 2024-12-10 at 01 15 50" src="https://github.com/user-attachments/assets/43c99d29-3a8d-433c-80ff-d36a431f5979">
